src/states/elder/mod.rs

```diff
@@ -1792,6 +1794,19 @@ impl Elder {
     pub fn identify_connection(&mut self, pub_id: PublicId, peer_addr: SocketAddr) {
         self.peer_map.identify(pub_id, peer_addr)
     }
+
+    pub fn send_msg_to_targets(
+        &mut self,
+        dst_targets: &[PublicId],
+        dg_size: usize,
+        message: Message,
+    ) {
+        self.send_message_to_targets(dst_targets, dg_size, message)
+    }
```

Comment:
Is this function necessary? Can't we just call `send_message_to_targets` directly wherever we're calling this function?

Reply:
The `send_message_to_targets` is defined under that `Base` trait.
Trying to use it means need to expose `Base` as public too, which incurs many comments to be added to those pub functions.  :(

Reply:
also it is for testing purpose only, so not really something we want in pub interface